app/views/errors/simple_404.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="site-content">
+	<div class="container">
+		<div class="site-content">
+			<div class="site-content-cap">
+					<h2 class="site-content-heading">
+					<% if flash[:error] %>
+						<% if flash[:error] = "Invalid Link" %>
+							<p id="error">Invalid Invitation Link! Please contact your organization.</p>
+						<% end %>
+						<% else %>
+						<p id="error">Page Could not be Found.</p>
+						<p>You may have mistyped the address or the page may have moved.</p>
+					<% end %>
+					</h2>
+			</div>
+		</div>
+	</div>
+</div>
```

Comment:
Also from my comment https://github.com/education/classroom/pull/510/files#r56814029 we can rename this to `not_found.html.erb`